packages/cli/test/integration.test.ts

```diff
@@ -352,24 +355,24 @@ test.after.always(async () => {
   }
 });
 
-test('default command should prompt login with empty auth.json', async t => {
+test('default command should prompt login with empty auth.json', async done => {
   await fs.writeFile(getConfigAuthPath(), JSON.stringify({}));
   try {
     await execa(binaryPath, [...defaultArgs]);
-    t.fail();
+    done.fail();
   } catch (err) {
-    t.true(
+    expect(
       err.stderr.includes(
         'Error: No existing credentials found. Please run `vercel login` or pass "--token"'
       )
-    );
+    ).toBe(true);
   }
 });
 
 // NOTE: Test order is important here.
 // This test MUST run before the tests below for them to work.
-test('login', async t => {
-  t.timeout(ms('1m'));
+test('login', async () => {
+  // NOTE: Needs timeout of 1m
```

Comment:
I'm going to leave these comments in place so that we know where the timeout increases need to be. Jest only lets us set this per file.

Reply:
I think jest lest you set a timeout per function as the third parameter like this

```js
test('foo', () => { expect(1).toBe(1) }, 5000)
```

But its probably fine to have it on the file instead